st2api/st2api/controllers/v1/rules.py

```diff
@@ -52,7 +52,12 @@ class RuleController(resource.ContentPackResourceController):
         'name': 'name',
         'pack': 'pack',
         'action': 'action.ref',
-        'trigger': 'trigger'
+        'trigger': 'trigger',
+        'enabled': 'enabled'
+    }
+
+    filter_transform_functions = {
```

Comment:
Done 31c3e226c56619c90d8fb09938f60164593beb34

Took the liberty of making this strict.